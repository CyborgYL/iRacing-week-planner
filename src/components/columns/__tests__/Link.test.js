import React from 'react';
import { shallow } from 'enzyme';
import { describe, test } from '@jest/globals';

import { Link } from '../index';

describe('components/columns/Link', () => {
  test('renders correctly', () => {
    const component = shallow(
      <Link
        race={{
          seasonId: 4234,
        }}
      />
    );

    expect(component).toMatchSnapshot();
    expect(component.find('a').first().prop('href')).toContain("4234");
  });
});
