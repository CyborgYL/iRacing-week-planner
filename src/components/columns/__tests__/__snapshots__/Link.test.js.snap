// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/columns/Link renders correctly 1`] = `
<td>
  <a
    href="http://members.iracing.com/membersite/member/SeriesSessions.do?season=4234"
    rel="noreferrer"
    target="_blank"
  >
    <LinkIcon />
  </a>
</td>
`;
